package.json

```diff
@@ -35,8 +35,9 @@
   "devDependencies": {
     "chai": "4.3.7",
     "chai-as-promised": "7.1.1",
-    "eslint": "8.46.0",
+    "eslint": "^8.46.0",
```

Comment:
Can you reinstall these packages with an exact version? All the other packages in this project are installed that way.